source/Halibut.TestUtils.CompatBinary.Base/DelegateEchoService.cs

```diff
@@ -29,7 +29,16 @@ public string SayHello(string name)
         public bool Crash()
         {
             Console.WriteLine("Forwarding Crash() call to delegate");
-            return echoService.Crash();
+            try
+            {
+                return echoService.Crash();
+            }
+            catch (Exception e)
```

Comment:
Just wondering, why did we wrap this?

Reply:
Otherwise its hard to determine what the actual service returned, we need to know the error message is a `InvocationTargetException`

Reply:
Just wondering, we can leave it as a string wrapper. Or would it be better making a custom exception, and throw that instead?

Reply:
The custom exception gets `toString()` called on it and then placed into the message section of the Exception sent over the wire bringing us back to square one.